src/common/base/src/utils/topic_util.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 RobustMQ Team
```

Comment:
It is recommended to add test cases to the methods in this file, because the methods in this file are the core functionality of this feature.

Reply:
okay, i'll make this change